tests/kubeless-test-client/k8s.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
+metadata:
+  namespace: kubeless-test
+  name: test-hello
+  labels:
+    function: test-hello
+spec:
+  authentication: []
+  hostname: test-hello.yfactory.sap.corp
+  service:
+    name: test-hello
+    port: 8080
+---
+apiVersion: gateway.kyma.cx/v1alpha2
+kind: Api
+metadata:
+  namespace: kubeless-test
+  name: test-event
+  labels:
+    function: test-event
+spec:
+  authentication: []
+  hostname: test-event.yfactory.sap.corp
+  service:
+    name: test-event
+    port: 8080
+---
+apiVersion: remoteenvironment.kyma.cx/v1alpha1
+kind: EventActivation
+metadata:
+  namespace: kubeless-test
+  name: test-event
+spec:
+  source:
+    namespace: test
+    type: test
+    environment: test
+---
+apiVersion: eventing.kyma.cx/v1alpha1
+kind: Subscription
+metadata:
+  namespace: kubeless-test
+  name: test-event
+spec:
+  endpoint: http://test-event.kubeless-test:8080/
```

Comment:
Do you really need Api / Ingress to expose your service if you call via internal fqdn ?

Reply:
Yes. In the test we want to ensure that the kubeless controller creates the function correctly and that it is available via api/ingress. This is short-lived and removed at the end of the test.